config/development.toml

```diff
@@ -766,3 +766,7 @@ card_networks = "Visa, AmericanExpress, Mastercard"
 
 [network_tokenization_supported_connectors]
 connector_list = "cybersource"
+
+[grpc_client.dynamic_routing_client] 
```

Comment:
can you also add it in `example.toml`?

Also, do we need to have this here in `development.toml`

Reply:
Already there